cfpb/cfgov-refresh
# Home page: Add H1

Adds hard-coded, visually hidden H1 to home page for accessibility purposes  (GHE 61)

## Additions

- H1 in home page
- translation for home page H1

## Testing

1. Check out branch
2. Navigate to [English](http://localhost:8000/?nhp=True) and [Spanish](http://localhost:8000/es?nhp=True) home pages and verify presence of hidden H1

## Screenshots

<img width="1157" alt="Screen Shot 2019-12-09 at 1 25 46 PM" src="https://user-images.githubusercontent.com/778171/70474434-71c71100-1a87-11ea-918d-5ffc686b666b.png">
